templates/colors/literals-swift3.stencil

```diff
@@ -2,19 +2,20 @@
 
 {% if palettes %}
 {% set enumName %}{{param.enumName|default:"ColorName"}}{% endset %}
+{% set accessModifier %}{% if param.publicAccess %}public{% else %}internal{% endif %}{% endset %}
```

Comment:
Although it's not necessary to mark these as `internal` per se, I think it's good to actually do it, as it clearly communicates this to the user/developer.

Reply:
Yeah that's why I took that approach instead of the one in #81 where it was either `public ` when on… and empty string when off.
I prefer the current approach both for balance in the template (as including the space in one case and nothing in the other and not having the space when using `{{accessModifier}}func` feels strange) and because making internal explicit feels more… well… explicit 😄